pkgs/top-level/all-packages.nix

```diff
@@ -37724,6 +37724,8 @@ with pkgs;
 
   zxing-cpp = callPackage ../development/libraries/zxing-cpp { };
 
+  nyan = callPackage ../development/libraries/nyan { };
```

Comment:
This file may not be entirely in alphabetical order, but there's no need to keep it at way. I'd recommend putting this in a more appropriate position.